packages/suite-desktop/src-electron/modules/auto-updater.ts

```diff
@@ -187,11 +194,16 @@ const init: Module = ({ mainWindow, store }) => {
             `- Last version: ${version}`,
             `- Last release date: ${releaseDate}`,
             `- Downloaded file: ${downloadedFile}`,
+            `- Release notes: ${releaseNotes}`,
```

Comment:
Do we need release notes here?

Reply:
9a984536c36bb4cfb7bb93709666a45af9227094